app/assets/stylesheets/darkswarm/home_tagline.css.scss

```diff
@@ -34,7 +34,7 @@
       max-width: 45%;
 
       @media all and (min-height: 500px) {
-        max-width: 80%;
```

Comment:
Hi @lilwillifo, thanks for the contribution :slightly_smiling_face: 
I wonder if the max-width in this media query for < 500px screens is set to `max-width: 80%` for a reason... did you experiment with this at different screen sizes?

Reply:
There's a bit of complexity in displaying these logos, in that each country has logo files that are slightly different widths, so we have to keep that in mind.